### PR TITLE
fix(amcharts5): add toggleSeries

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
Toggle series when click toggle circle in widget data table.

### Things to Talk About
There's some style bugs in chart and data table.
I'll fix it, soon.

### CODE REVIEW

File: src/common/composables/amcharts5/concepts-helper.ts [hunk 1]
```diff
@@ -67,3 +68,27 @@ export const createBullet = (root: Root, settings: IBulletSettings): am5.Bullet
 export const createCircle = (root: Root, settings: ICircleSettings, circleTemplate: am5.Template<Circle>): am5.Circle => am5.Circle.new(root, {
     ...settings,
 }, circleTemplate);
+
+export const toggleSeries = (chart: am5.SerialChart, index: number) => {
```
Comment (yuda110) [inline src/common/composables/amcharts5/concepts-helper.ts:72]: This function is copied from amcharts/helper.

File: src/common/composables/amcharts5/concepts-helper.ts [hunk 2]
```diff
@@ -67,3 +68,27 @@ export const createBullet = (root: Root, settings: IBulletSettings): am5.Bullet
...
+    if (chart instanceof am5percent.PieChart) {
+        const series = chart.series?.getIndex(0);
+        if (!series) return;
+
+        const slice = series.slices.values[index];
+        if (!slice || !slice.dataItem) return;
+
+        if (slice.isHiding() || slice.isHidden()) {
```
Comment (minseolee) [inline src/common/composables/amcharts5/concepts-helper.ts:80]: Is `isHiding` and `isHidden` same thing ? 🤔

**Review by minseolee — APPROVED** [review]
LGTM ! !

File: src/common/composables/amcharts5/concepts-helper.ts (hunk 2 above)
Comment (yuda110) [reply src/common/composables/amcharts5/concepts-helper.ts:80]: [amcharts tutorial](https://www.amcharts.com/docs/v4/tutorials/toggling-series-via-api/#Toggling) recommends using both.

Comment (minseolee) [reply src/common/composables/amcharts5/concepts-helper.ts:80]: I'll note it. Thanks a lot.

File: src/services/dashboards/widgets/_components/WidgetDataTable.vue [hunk 3]
```diff
@@ -326,7 +326,7 @@ export default defineComponent<Props>({
         /* event */
         const handleClickLegend = (index) => {
             // if (props.printMode) return;
-            emit('toggle-legend', props.legends[index]);
+            emit('toggle-legend', index);
```
Comment (sulmoJ) [inline src/services/dashboards/widgets/_components/WidgetDataTable.vue:329]: 👍 👍

**Review by sulmoJ — APPROVED** [review]
LGTM!!

**Pull request merged**